src/oqsconfig.h.cmake

```
@@ -1,6 +1,10 @@
 // SPDX-License-Identifier: MIT
 
 #cmakedefine OQS_VERSION_TEXT "@OQS_VERSION_TEXT@"
+#define OQS_VERSION_MAJOR @OQS_VERSION_MAJOR@
+#define OQS_VERSION_MINOR @OQS_VERSION_MINOR@
+#define OQS_VERSION_PATCH @OQS_VERSION_PATCH@
+#define OQS_VERSION_PRE_RELEASE "@OQS_VERSION_PRE_RELEASE@"
```

```suggestion
#cmakedefine OQS_VERSION_PRE_RELEASE "@OQS_VERSION_PRE_RELEASE@"
```
If we use cmakedefine for this variable, we can do an ifdef check when printing the version string: see suggestion on `tests/system_info.c`